app/services/api/v1/message_service.rb

```diff
@@ -11,7 +11,20 @@ def create
         conversation = Conversation.find(@conversation_id)
         raise InvalidConversationError unless conversation.user1_id == @current_user.id || conversation.user2_id == @current_user.id
 
-        Message.create!(content: @message_params[:content], user: @current_user, conversation_id: @conversation_id)
+        @message = Message.create!(content: @message_params[:content], user: @current_user, conversation_id: @conversation_id)
+        notify
+        @message
+      end
+
+      def notify
+        conversation = Conversation.find(@conversation_id)
+        user1 = User.find(conversation.user1_id)
+        user2 = User.find(conversation.user2_id)
```

Comment:
You can wrap this logic into a method in `Conversation` model and here just call it.

```suggestion
        user  = Conversation.recipient_user
```